### PR TITLE
feat(git): Adding validation of pull requests title

### PR DESCRIPTION
**Pull request by Dracks**

### CODE REVIEW

**Review by HunterNN — APPROVED** [review]

**Comment by Darkbladecr** [comment]
Tried playing around with the workflow but for some reason the image is always coming back as not accessible? @Dracks 
```
Run amannn/action-semantic-pull-request@v5
Error: Resource not accessible by integration
```

**Comment by Dracks** [comment]
> Tried playing around with the workflow but for some reason the image is always coming back as not accessible? @Dracks
> 
> ```
> Run amannn/action-semantic-pull-request@v5
> Error: Resource not accessible by integration
> ```

yes, I saw it, is weird, Maybe we can create it from the marketplace https://github.com/amannn/action-semantic-pull-request

If you go to actions you will find it. 

Maybe the problem is that this is on my repo, and executed on my repo? (Maybe you can try to have it in a branch of codename-recon repository)

**Comment by Dracks** [comment]
it feels to me, is some kind of permission, maybe on the github action, since I don't have admin permissions of the repo, can we do some tests/checks with this? https://github.com/orgs/community/discussions/60820#discussioncomment-6445895

File: .github/workflows/ci_pr2.yaml
Comment (Dracks) [inline .github/workflows/ci_pr2.yaml:1]: this is the sample that they have in the action repo.

**Comment by HunterNN** [comment]
Screenshot_20240122_225428
Changed the permission, but still generates an error.

**Comment by Dracks** [comment]
I think only read repository will be needed.

**Comment by HunterNN** [comment]
I thought that was the part you wished for me to change?

**Comment by Dracks** [comment]
> I thought that was the part you wished for me to change? Or do you mean only the last option was needed?

Tbh, I don't have any idea, I don't know what was the default config, and at the end is try and error, right now. 

Can you check this config and try again (I expect is disabled, lets try to enable and see if that works)? 
image

At the end, in my company, everybody works directly over the repository and we create the P/Rs inside the repo, since this is a cross-repo, maybe that's the problem.

**Pull request closed without merging**